docs/api/tracker.rst

```diff
@@ -7,6 +7,21 @@
 Tracker
 =======
 
+.. edit-link::
+
+Trackers maintain the state of a dialogue between the assistant and the user in the form
+of conversation sessions. The default conversation session configuration looks as
+follows:
+
+.. code-block:: yaml
+
+  session_config:
+    session_expiration_time: 60  # value in minutes, 0 means infinitely long
```

Comment:
how about:
```suggestion
    session_expiration_time: 60  # value in minutes, 0 disables sessions
```

Reply:
not really, because the initial `action_session_start` is still run